src/library/dlganalysis.cpp

```diff
@@ -153,33 +148,25 @@ void DlgAnalysis::analysisActive(bool bActive) {
     }
 }
 
-// slot
-void DlgAnalysis::trackAnalysisFinished(int size) {
-    qDebug() << "Analysis finished" << size << "tracks left";
-    if (size > 0) {
-        m_currentTrack = m_tracksInQueue - size + 1;
-    }
-}
-
-// slot
-void DlgAnalysis::trackAnalysisProgress(int progress) {
-    if (m_bAnalysisActive) {
-        QString text = tr("Analyzing %1/%2 %3%").arg(
-                QString::number(m_currentTrack),
-                QString::number(m_tracksInQueue),
-                QString::number(progress));
-        labelProgress->setText(text);
+void DlgAnalysis::slotAnalyzerQueueProgress(
+        int analyzerProgress, int finishedCount, int totalCount) {
```

Comment:
As I suggested above, you might take my implementation that shows one percentage for each thread rather than an average value.